wiki/Contests/Mapping_Olympiad/en.md

```diff
@@ -0,0 +1,83 @@
+# osu! Mapping Olympiad
+
+![](https://osu.ppy.sh/wiki/shared/news/banners/osu_mapping_olympiad.jpg)
+
+**osu! Mapping Olympiad** is an all-gamemode official contest line-up that was originally from a project of official beatmapping contests revival in 2018. Its particularity is to explore different contest concepts, going from traditional mapping contests to speedmapping or aspire, to mention just a few. 
+
+## Contest hosts
+
+| Game mode | User |
+| :-- | :-- |
+| ![osu!](/wiki/shared/mode/osu.png) osu! | ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870), ![][flag_FR] [Imakuri](https://osu.ppy.sh/users/6100837), ![][flag_FR] [Pachiru](https://osu.ppy.sh/users/2850983) |
+| ![osu!taiko](/wiki/shared/mode/taiko.png) osu!taiko | ![][flag_CA] [Gabe](https://osu.ppy.sh/users/654108) |
+| ![osu!catch](/wiki/shared/mode/catch.png) osu!catch | ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) |
+| ![osu!mania](/wiki/shared/mode/mania.png) osu!mania | ![][flag_FR] [Pachiru](https://osu.ppy.sh/users/2850983), ![][flag_GB] [Asherz007](https://osu.ppy.sh/users/9014047) |
+
+## Ruleset
+
+*Note: Keep in mind that the ruleset may be subject to change depending on editions' restrictions.*
+
+- Each entry must follow the [Ranking Criteria](/wiki/Ranking_Criteria). Entries found to break the Ranking Criteria will be subject to penalties, ranging from point deductions to disqualification based on severity.
+- Beatmap submissions must not exceed 15MB.
+- Storyboards will not be considered in the judging process.
+- Custom [hitsounds](/wiki/Beatmapping/Hitsound) may be used.
+- Collaboration on entries is not permitted.
+- Maps submitted to the contest must not be uploaded using the [Beatmap Submission System](/wiki/Submission) (BSS) until the contest is concluded and the winners are announced.
+
+## Previous contests
+
+The leaderboards right below display the top 3 contestants from each iteration since the first edition of the osu! Mapping Olympiad.
+
+| Contest listing | ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | Results post |
+| :-- | :-- | :-- | :-- | :-- |
```

Comment:
crowns look better when centered, no? unfortunately that realigns the whole column, so feel free to discard it if you don't like the new look
```suggestion
| :-- | :-: | :-: | :-: | :-- |
```

Reply:
I prefer to keep it like it is cause it looks quite weird with everything centered